PowerPoint Unzipped File Experiments/OneImageAndPlainText - Readable XML Formatting.pptx/docProps/core.xml

```diff
@@ -1,9 +1,9 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why was the standalone removed? Do you know what that was for in the first place?

Reply:
The XML formatting tool in ATOM removes it. But this was in the "experimental" directory. Doesn't affect our code.